helm/cluster-api-provider-azure/values.yaml

```diff
@@ -0,0 +1,19 @@
+name: cluster-api-azure-controller
+image:
+  registry: gcr.io
+  name: k8s-staging-cluster-api-azure/cluster-api-azure-controller
+  tag: v0.4.12
```

Comment:
Should we depend on the current `master` commit sha so we can add the CLI parameter to use a label to filter CRs?

Reply:
I went with the latest `v1alpha3` release here, since other CAPI apps also went used `v1alpha3` controllers.
Just to start with something, until we decide how we're gonna do.